uefi/src/lib.rs

```diff
@@ -41,6 +41,7 @@
 #![feature(maybe_uninit_slice)]
 #![feature(negative_impls)]
 #![feature(ptr_metadata)]
+#![feature(error_in_core)]
```

Comment:
As discussed in another PR, we're going to want to gate new unstable features behind an `unstable` feature in the UEFI crate: https://github.com/rust-osdev/uefi-rs/pull/584/files#r1031848552

However, we haven't merged in the new feature gate yet, so I think I'll go ahead and merge this PR, and update https://github.com/rust-osdev/uefi-rs/pull/590 to put this behind the `unstable` fate.